pkg/doc/verifiable/credential.go

```diff
@@ -201,10 +201,10 @@ type Issuer struct {
 type Subject interface{}
 
 // CredentialStatus defines status of Verifiable Credential
-type CredentialStatus typedID
+type CredentialStatus TypedID
 
 // CredentialSchema defines a link to data schema which enforces a specific structure of Verifiable Credential.
-type CredentialSchema typedID
+type CredentialSchema TypedID
```

Comment:
Is this going to cause a shadowing, since the name of this variable matched the name of the type "TypedID". May want to change this name

Reply:
Good catch @DRK3 ! I cleaned this in other places but miss those assignments.

Reply:
Gotcha! Thanks